pkgs/development/python-modules/gremlinpython/default.nix

```diff
@@ -53,26 +56,40 @@ buildPythonPackage rec {
     pytestCheckHook
     mock
     pyhamcrest
+    pyyaml
     radish-bdd
   ];
 
   # disable custom pytest report generation
   preCheck = ''
     substituteInPlace setup.cfg --replace 'addopts' '#addopts'
-    export TEST_TRANSACTIONS='false'
   '';
 
   # many tests expect a running tinkerpop server
   disabledTestPaths = [
     "tests/driver/test_client.py"
     "tests/driver/test_driver_remote_connection.py"
+    "tests/driver/test_driver_remote_connection_http.py"
     "tests/driver/test_driver_remote_connection_threaded.py"
+    "tests/driver/test_web_socket_client_behavior.py"
     "tests/process/test_dsl.py"
     "tests/structure/io/test_functionalityio.py"
   ];
-  pytestFlagsArray = [
+  pytestFlagsArray = let
+    # more tests that expect a running tinkerpop server
+    transactionTests = [
+      "test_transaction_commit"
+      "test_transaction_rollback"
+      "test_transaction_no_begin"
+      "test_multi_commit_transaction"
+      "test_multi_rollback_transaction"
+      "test_multi_commit_and_rollback"
+      "test_transaction_close_tx"
+      "test_transaction_close_tx_from_parent"
+    ];
```

Comment:
Why does `disabledTests` not work for these?

Reply:
It does, but there are similarly named tests in different files which I'd rather avoid disabling too.